### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 1 - Esercizio 10

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#475] [Esercizi M-Z]
![foglio 1 es.10](https://github.com/user-attachments/assets/cb0e8e59-e14f-4b15-9781-ab545eb7ab6f)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/e3c1ec0b-7a67-46c9-8adc-19fa90529cda)
